ros_msft_onnx/src/yolo_box.cpp

```diff
@@ -142,17 +160,17 @@ namespace yolo
 
                     float x = ((float)cx + Sigmoid(tx)) * CELL_WIDTH;
                     float y = ((float)cy + Sigmoid(ty)) * CELL_HEIGHT;
-                    float width = (float)exp(tw) * CELL_WIDTH * anchors[b * 2];
-                    float height = (float)exp(th) * CELL_HEIGHT * anchors[b * 2 + 1];
+                    float width = (float)exp(tw) * CELL_WIDTH * _anchors[b * 2];
```

Comment:
Since the number of anchors is now dynamic, we should validate this before indexing into it.